packages/lit-dev-content/site/tutorials/content/first-component/06.md

```diff
@@ -0,0 +1,140 @@
+In this section we'll make it so when you click on the `word-viewer` the
+animation plays in reverse.
+
+Before adding the event handling we need a new piece of internal state to
+represent the current animation `playDirection`.
+
+{% switchable-sample %}
+
+```ts
+// word-viewer.ts
+
+  @state() private playDirection = 1;
+
+```
+
+```js
+// word-viewer.js
+
+  static properties = {
+    playDirection: {state: true},
+    ...
+  };
+
+  constructor() {
+    ...
+    this.playDirection = 1;
+  }
+
+```
+
+{% endswitchable-sample %}
+
+Next the math in `render()` to find the `word` needs to be updated to account
+for a negative `this.idx`.
+
+{% switchable-sample %}
+
+```ts
+// word-viewer.ts
+
+  render() {
+    ...
+    const word = splitWords[((this.idx % splitWords.length) + splitWords.length) % splitWords.length];
+    ...
+  }
+
+```
+
+```js
+// word-viewer.js
+
+  render() {
+    ...
+    const word = splitWords[((this.idx % splitWords.length) + splitWords.length) % splitWords.length];
+    ...
+  }
+
+```
+
+{% endswitchable-sample %}
+
+Change `tickToNextWord` to choose the next word based on the `playDirection`.
+
+{% switchable-sample %}
+
+```ts
+// word-viewer.ts
+
+  tickToNextWord = () => { this.idx += this.playDirection; };
+
+```
+
+```js
+// word-viewer.js
+
+  tickToNextWord = () => { this.idx += this.playDirection; };
+
+```
+
+{% endswitchable-sample %}
+
+And add a `switchPlayDirection` method to invert the  `playDirection`.
+
+
+{% switchable-sample %}
+
+```ts
+// word-viewer.ts
+
+  switchPlayDirection() {
+    this.playDirection *= -1;
+  }
+
+```
+
+```js
+// word-viewer.js
+
+  switchPlayDirection() {
+    this.playDirection *= -1;
+  }
+
+```
+
+{% endswitchable-sample %}
+
+Finally, invoke `this.switchPlayDirection` when the `<pre>` element is clicked
```

Comment:
hmm maybe we should focus on "event" listening and say something along the lines of:

> when the `click` even is dispatched by the `pre` element... on user click?

does it sound too confusing? too verbose?

Reply:
A bit confused how to address this comment exactly. Slightly re-worded.